build.gradle

```diff
@@ -1082,7 +1082,7 @@ subprojects {
         }
 
         // Create a task for each JUnit test class whose name is the same as the JUnit class name.
-        sourceSets.test.allJava.filter { it.path.contains('test/junit') }.forEach { file ->
+        sourceSets.test.allJava.filter { it.path.matches('.*test[\\/]+junit.*') }.forEach { file ->
```

Comment:
What does the `[\\/]+` mean? That there can be one or more slashes? Why is that useful?
What path separator do you see on Windows?
Would `.contains("test\\/junit")` do the same thing?

Reply:
maybe I will change it to [\\/]